src/averageAges.js

```diff
@@ -14,12 +14,22 @@
  *
  * @return {number}
  */
+
+const calculateAverage = (arr) => arr.reduce((a, b) => a + b, 0) / arr.length;
+const calculateLiveLength = (people) => (
+  people.map(person => person.died - person.born)
+);
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const diedCentury = (person) => (
+    !century || Math.ceil(person.died / 100) === century
+  );
+
+  const men = people.filter((person) => (
+    person.sex === 'm' && diedCentury(person)
+  ));
```

Comment:
it's interesting approach 🚀
However I think it would be more readable if you used a ternary operator here instead to perfom this check, in this wait it's not so obvious at the first glance what it does and verify the conditions